environment.yml

```diff
@@ -17,4 +17,4 @@ dependencies:
     - deepdish
     - SimPEG
     - empymod
-    - git+https://github.com/geoscixyz/geosci-labs
+    - git+https://github.com/wwzzyyzzrr/geosci-labs
```

Comment:
could you please revert this change? the environment.yml should point to the main repository.

Reply:
In the future, you can just create a branch on this repo rather than using a fork and that should simplify things a bit. You should now have the ability to push to this repo since you are a geoscixyz team member 🎉